holidays/countries/egypt.py

```diff
@@ -42,54 +47,54 @@ def _populate(self, year):
         super()._populate(year)
 
         # New Year's Day
-        self._add_new_years_day("New Year's Day - Bank Holiday")
+        self._add_new_years_day(tr("رأس السنة الميلادية - عطلة بنوك"))
 
         # Coptic Christmas
-        self._add_christmas_day("Coptic Christmas")
+        self._add_christmas_day(tr("عيد الميلاد المجيد (تقويم قبطي)"))
 
         # 25th of Jan
         if year >= 2012:
-            self._add_holiday("Revolution Day - January 25", JAN, 25)
+            self._add_holiday(tr("عيد ثورة 25 يناير"), JAN, 25)
```

Comment:
As well as "January 25" unless it's a part of the official holiday name.

Reply:
It's part of the official holiday name I believe.
[Reference](https://www.presidency.eg/en/%D9%85%D8%B5%D8%B1/%D8%A7%D9%84%D8%B9%D8%B7%D9%84%D8%A7%D8%AA-%D8%A7%D9%84%D8%B1%D8%B3%D9%85%D9%8A%D8%A9/)